docs/contributtion/feature-flags.md

```diff
@@ -0,0 +1,67 @@
+# Feature Flags
+
+## Table of Contents
+
+- [Current feature flags](#current-feature-flags)
+- [Turning on feature flags in SASS](#turning-on-feature-flags-in-sass)
+
+The Spirit Design System codebase ships with a number of feature flags. These feature
+flags enable new behavior and styling, allowing you to opt-in to new (and
+sometimes breaking) changes while using the current version. A feature flag may
+be configured in React, SASS, or both.
+
+This section in the Storybook documents each feature flag that is available and
+how to configure it. Folders and stories within this section in Storybook show
+components with all feature flags turned on.
+
+## Current feature flags
+
+Feature flags prefixed with `enable-*` contain features we'd like consuming projects to
+test and give us feedback on. It will be made by default in the version specified in the name.
+In most of the cases these features will be breaking changes - logically or visually.
+They're generally stable and unlikely to change but may change based on your feedback.
+We just do not want to break your application or design without your consent.
```

Comment:
:D with consent we can break their app? :D

Reply:
In this case, the feature flag itself is the consent, e. g. you use that feature flag and set it on so this is considered as a consent, isn't it?